apps/src/blockly/addons/cdoFieldButton.ts

```diff
@@ -7,13 +7,13 @@ interface ColorOverrides {
 }
 
 interface CdoFieldButtonOptions {
-  value: string | undefined;
-  validator: FieldValidator<string> | null;
+  value?: string;
```

Comment:
This feels like a semantic difference? Previously, we were saying that value will be a string or undefined, and now we're saying it's a string but it's optional

Reply:
The reason I changed it is because we were already treating these as optional when initializing `FieldButton`. For example [here](https://github.com/code-dot-org/code-dot-org/blob/c6abf21d2cbfecb0cf94cef22e7708711cc92d53/apps/src/blockly/addons/cdoUtils.js#L339) we aren't passing value, validator, colorOverrides or allowReadOnlyClick. In order to make the compiler happy I had to make it optional.